cmd/main.go

```diff
@@ -36,7 +44,31 @@ var (
 )
 
 var saramaLogger = log.New(io.Discard, "[Sarama] ", log.LstdFlags)
+
+func InitTracer(JaegerExporterTracing bool) *sdktrace.TracerProvider {
+	resources, _ := resource.New(context.Background(),
+		resource.WithFromEnv(), // pull attributes from OTEL_RESOURCE_ATTRIBUTES and OTEL_SERVICE_NAME environment variables
+		resource.WithProcess(), // This option configures a set of Detectors that discover process information
+	)
+	//Could not use OTEL_TRACES_EXPORTER https://github.com/open-telemetry/opentelemetry-go/issues/2310
+	var exporter sdktrace.SpanExporter
+	if JaegerExporterTracing {
```

Comment:
Rather than passing the flag in could we read `OTEL_TRACES_EXPORTER` here and load `jaeger` or `otlptracegrpc` as appropriate based on the value of the env var?  

Obviously it would be nicer if https://github.com/open-telemetry/opentelemetry-go/issues/2310 was implemented, on that note maybe it would be better to have our own var for it to avoid conflicts in the future if #2310 gets traction.

Reply:
now if jaeger env variables are defined the exporter is jaeger else it ins otlp